.travis.yml

```diff
@@ -22,12 +22,12 @@ notifications:
   email:
     on_success: never
     on_failure: change
-deploy:
-  provider: heroku
-  buildpack: scala
-  api_key:
-    secure: NE6m0300BPmc8jJQbpTFyOP1uU6uK+7aVpN5lmKAPhV2gByzzq9XT3uwYqPcD9turCYujDXAmXrBzI6ISfLlrqpcxKjSupeUvvjpOxqhbpP65hA79VTuxDIajazzXrvjuKPgvUJAjD+1qdQWzoOjKJ4Y/iJ1YL+fqt8omOSFda0=
-  app: sphere-sunrise
-  on:
-    repo: sphereio/sphere-sunrise
-    branch: master
+#deploy:
```

Comment:
I would not outcomment stuff, it is dead code. You still can get it from history.